src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -814,6 +814,14 @@ private async Task<Dictionary<string, ConfigurationSetting>> LoadSelected(
                         LabelFilter = loadOption.LabelFilter
                     };
 
```

Comment:
Do we need to setup tag filters here: https://github.com/Azure/AppConfiguration-DotnetProvider/blob/87f0f85ca2e4011f82d93a864e35c6c804cd6c39/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs#L343-L348

Reply:
yes, will update